SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -35,6 +35,13 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 public:
 	virtual void FinishDestroy() override;
 	void DestroyConnection();
+	
+	using LoginTokenResponseCallback = TFunction<bool(const Worker_Alpha_LoginTokensResponse*)>;
+    
+    /// Register a callback using this function.
+    /// It will be triggered when receiving login tokens using the development authentication flow inside SpatialWorkerConnection.
+    /// @param cb - callback function.
+	void RegisterOnLoginTokensCb(const LoginTokenResponseCallback& Callback) {LoginTokenResCallback = Callback;}
```

Comment:
Nit - I'm still not in favour of using terms like Cb for callback.
If you can only have one callback then Set might be better than Register. I think I missed that there was only one before.

Reply:
Already modified `Cb` to `Callback`.
As for `set` or `register` , I feel it might need some other callback in `OnPlayerIdentityToken` or `OnConnectionSuccess` if necessary. So using `register` should be fine in current situation.